src/Core/Resolvers/MySqlQueryBuilder.cs

```diff
@@ -179,18 +177,25 @@ public override string BuildForeignKeyInfoQuery(int numberOfParameters)
         /// <summary>
         /// Makes the query segments to store PK during an update
         /// </summary>
-        private (string, string, string) MakeStoreUpdatePK(List<string> columns,
-                                                           List<LabelledColumn> outputColumns)
+        private (string, string, string) MakeStoreUpdatePK(BaseSqlQueryStructure structure, List<LabelledColumn> outputColumns)
```

Comment:
Comment needs to be updated to include description of what the return tuple represents.

Reply:
Comment should also address:
- why this is necessary for MySQL only
- example of a built segment for sets, updates, select

Reply:
Function name should be made more intuitive e.g. `MakeQuerySegmentsForUpdate`

Reply:
Yep, I'll make those additional changes.

Reply:
> why this is necessary for MySQL only

Do you mean why this function is called only for MySql or why are we not including the read-only column in the set/update/select clause? I am assuming the latter.

Reply:
Clarified and added example.